g2p_payment_phee/models/payment_manager.py

```diff
@@ -0,0 +1,191 @@
+            # vals.append((Command.LINK, payment.id))
+            vals.append(Command.link(payment.id))
+            payments_to_add_ids.append(payment.id)
+        if payments_to_add_ids:
+            # Create payment batch
+            if self.create_batch:
+                new_batch_vals = {
+                    "cycle_id": cycle.id,
+                    "payment_ids": vals,
+                    "stats_datetime": fields.Datetime.now(),
+                }
+                batch = self.env["g2p.payment.batch"].create(new_batch_vals)
+                # Update processed payments batch_id
+                self.env["g2p.payment"].browse(payments_to_add_ids).update(
+                    {"batch_id": batch.id}
+                )
+
+            kind = "success"
+            message = _("%s new payments was issued.") % len(payments_to_add_ids)
+            links = [
+                {
+                    "label": "Refresh Page",
+                }
+            ]
+        else:
+            kind = "danger"
+            message = _("There are no new payments issued!")
+            links = [{}]
+
+        return {
+            "type": "ir.actions.client",
+            "tag": "display_notification",
+            "params": {
+                "title": _("Payment"),
+                "message": message + " %s",
+                "links": links,
+                "sticky": True,
+                "type": kind,
+            },
+        }
+
+    def send_payments(self, batches):
+        # Bulk Transfer to PHEE API
+        payment_endpoint_url = self.payment_endpoint_url
+        tenant_id = self.tenant_id
+        _logger.info(
+            f"DEBUG! send_payments Manager: PHEE - URL: {payment_endpoint_url} tenant: {tenant_id}"
+        )
+        for rec in batches:
+            # TODO: determine the appropriate filename
+            filename = "test-payload.csv"
+            bulk_trans_url = f"{payment_endpoint_url}/{rec.name}/{filename}"
+            headers = {
+                "Platform-TenantId": tenant_id,
+            }
+            data = "id,request_id,payment_mode,account_number,amount,currency,note\n"
+            for row in rec.payment_ids:
+                # TODO: Get data for payment_mode and account_number
+                payment_mode = "slcb"
+                account_number = "SE0000000000001234567890"
+                data += f"{row.id},{rec.name},{payment_mode},{account_number},"
+                data += f"{row.amount_issued},{row.currency_id.name},{row.partner_id.name}\n"
+
+            try:
+                res = requests.post(
+                    bulk_trans_url, headers=headers, data=data, verify=False
+                )
+                res.raise_for_status()
+                # access JSOn content
+                jsonResponse = res.json()
+                _logger.info(f"PHEE API: jsonResponse: {jsonResponse}")
+                for key, value in jsonResponse.items():
```

Comment:
Add a `# TODO: Perform detailed parsing of jsonResponse`

Reply:
Done